### PR TITLE
Update Cirrus Images + follow-up fixes

### PR DESCRIPTION
**Pull request by vrothberg**
Continuation of https://github.com/containers/oci-seccomp-bpf-hook/pull/67

### CODE REVIEW

**Comment by vrothberg** [comment]
Alright, it seems related to cgroupsv2 (and maybe crun). Need to wait for Giuseppe to have a look.

**Comment by vrothberg** [comment]
@giuseppe @rhatdan ... I think we need to revive getting this done. The problem is that the hook is being killed during container clean up. It stopped working at some point. Maybe we never hit the race before and were lucky, or there's more to it.

**Comment by rhatdan** [comment]
Agreed.

File: .cirrus.yml [hunk 1]
```diff
@@ -10,9 +10,9 @@ env:
     HOME: "/root"  # not set by default
     GOCACHE: "${HOME}/.cache/go-build"
 
-    # VM Images are maintained in the libpod repo.
-    _BUILT_IMAGE_SUFFIX: "libpod-6224667180531712"  # From the packer output of 'build_vm_images_script'
-    FEDORA_CACHE_IMAGE_NAME: "fedora-32-${_BUILT_IMAGE_SUFFIX}"
+    # VM Image built in containers/automation_images
+    _BUILT_IMAGE_SUFFIX: "c6110627968057344"
```
Comment (cevich) [inline .cirrus.yml:14]: @vrothberg might try using "c6233039174893568" for F33  (above is for F32 IIRC)

**Pull request merged**